packages/syft/tests/syft/core/smpc/protocol/beaver/beaver_test.py

```diff
@@ -0,0 +1,26 @@
+# syft absolute
+from syft.core.smpc.store import CryptoPrimitiveProvider
+
+
+def test_beaver_mul(get_clients) -> None:
```

Comment:
Passthrough Tensor had an error , where it automatically converts matmul ops to np.int64 , which raised an error, so I omitted matmul, but I tested manually,